google/cloud/bigtable/admin/integration_tests/admin_backup_integration_test.cc

```diff
@@ -37,102 +38,129 @@ using ::google::protobuf::util::TimeUtil;
 using ::testing::Contains;
 using ::testing::Not;
 namespace btadmin = ::google::bigtable::admin::v2;
-namespace bigtable = ::google::cloud::bigtable;
 
 class AdminBackupIntegrationTest
     : public bigtable::testing::TableIntegrationTest {
```

Comment:
I think this means you are still using the hand-crafted clients for the base class `SetUp()` and cleanup. Consider a future PR to switch the base classes to use the new admin APIs.

Reply:
Yes. The plan is to convert the `TableIntegrationTest` implementation in the next PR as part of the "prefer generated APIs" issue. I edited the PR description to reflect that, but probably after you had looked at the code.